xchange-examples/src/main/java/org/knowm/xchange/examples/blockchain/BlockchainAddressDemo.java

```diff
@@ -1,32 +1,134 @@
+  }
+
+  private static void accountServiceDemo() throws IOException {
+    AccountService accountService = BLOCKCHAIN_EXCHANGE.getAccountService();
+
+    System.out.println("===== withdrawFunds =====");
+    String withdraw = accountService.withdrawFunds(Currency.BTC, BigDecimal.valueOf(0.00005), "ea1f34b3-e77a-4646-9cfa-5d6d3518c6d3");
+    System.out.println(objectMapper.writeValueAsString(withdraw));
+
+    System.out.println("===== requestDepositAddress =====");
+    String address = accountService.requestDepositAddress(Currency.ETH);
+    System.out.println(objectMapper.writeValueAsString(address));
+
+    System.out.println("===== requestDepositAddressData =====");
+    AddressWithTag addressWithTag = accountService.requestDepositAddressData(Currency.ETH);
+    System.out.println(objectMapper.writeValueAsString(addressWithTag));
+
+    System.out.println("===== getFundingHistory =====");
+    TradeHistoryParams params = accountService.createFundingHistoryParams();
 
-    BitcoinAddress bitcoinAddress = blockchain.getBitcoinAddress("XXX");
-    System.out.println(bitcoinAddress.toString());
+    if (params instanceof TradeHistoryParamsTimeSpan) {
+      final TradeHistoryParamsTimeSpan timeSpanParam = (TradeHistoryParamsTimeSpan) params;
+      timeSpanParam.setStartTime(
+              new Date(System.currentTimeMillis() - (1 * 12 * 30 * 24 * 60 * 60 * 1000L)));
+    }
+    if (params instanceof HistoryParamsFundingType) {
+      ((HistoryParamsFundingType) params).setType(FundingRecord.Type.DEPOSIT);
+    }
 
-    BitcoinAddresses bitcoinAddresses = blockchain.getBitcoinAddresses("XXX");
-    for (BitcoinAddress bitcoinAddress2 : bitcoinAddresses.getBitcoinAddresses()) {
-      System.out.println(bitcoinAddress2.toString());
+    List<FundingRecord> fundingDepositsRecords = accountService.getFundingHistory(params);
+    System.out.println(objectMapper.writeValueAsString(fundingDepositsRecords));
+
+    if (params instanceof HistoryParamsFundingType) {
+      ((HistoryParamsFundingType) params).setType(FundingRecord.Type.WITHDRAWAL);
     }
+
+    List<FundingRecord> fundingWithdrawalRecords = accountService.getFundingHistory(params);
+    System.out.println(objectMapper.writeValueAsString(fundingWithdrawalRecords));
+
+    System.out.println("===== getDynamicTradingFees =====");
+    Map<CurrencyPair, Fee> tradingFees = accountService.getDynamicTradingFees();
+    System.out.println(objectMapper.writeValueAsString(tradingFees));
+
+  }
+
+  private static void tradeServiceDemo() throws IOException {
+    TradeService tradeService = BLOCKCHAIN_EXCHANGE.getTradeService();
+
+    LimitOrder order = new LimitOrder.Builder(Order.OrderType.ASK, new CurrencyPair(Currency.USDT, Currency.USD))
+            .limitPrice(BigDecimal.valueOf(1.0))
+            .originalAmount(BigDecimal.valueOf(5.0))
+            .build();
+
+    System.out.println("===== placeLimitOrder =====");
+//    String limitOrderId = tradeService.placeLimitOrder(order);
```

Comment:
should we remove it or make it work?

Reply:
It was removed, only the methods for accountservice remained.